OpenEdXMobile/src/main/java/org/edx/mobile/view/custom/URLInterceptorWebViewClient.java

```diff
@@ -89,6 +98,9 @@ private void setupWebView(WebView webView) {
         //We need to hide the loading progress if the Page starts rendering.
         webView.setWebChromeClient(new WebChromeClient() {
             public void onProgressChanged(WebView view, int progress) {
+                if (progress > 25) {
```

Comment:
The magic number `25` should be explained as to why it has been added here and what is its purpose.